test/loader/test_cache.py

```diff
@@ -0,0 +1,29 @@
+import torch
+
+from torch_geometric.data import Data
+from torch_geometric.loader import CachedLoader, NeighborLoader
+from torch_geometric.testing import withPackage
+
+
+@withPackage('pyg_lib')
+def test_cached_loader():
```

Comment:
Just for reminder, we should also have a test case for `make_batches_cacheable` once finalised.

Reply:
Tests of `make_batches_cacheable` are located in `test_basic_gnn.py` file. 😉